code/game/machinery/slotmachine.dm

```diff
@@ -45,13 +49,13 @@
 
 	toggle_reel_spin(0)
 
-	for(var/cointype in typesof(/obj/item/coin))
+	for(cointype in typesof(/obj/item/coin))
```

Comment:
this change doesn't make sense can you explain?

Reply:
Since I kept the cointype on other areas of the code, I initialized the var at the start, so now it doesn't need to be initialized again, just used

Reply:
then why does this line exist

Reply:
To store the values of every coin in an array, to be used when doing payout (Honestly it is actually weird, since it only pays silver and iron, which are 10 and 1)

Reply:
ok I see what's going on now.